test/utils.test.js

```diff
@@ -659,6 +660,13 @@ describe('processPackage', () => {
     expect(entities).toMatchObject(JSON.parse(fs.readFileSync('/basic_manifest_res_namesonly.json')))
   })
 
+  test('basic manifest with package parameters', async () => {
```

Comment:
Add a similar test for multi packages, where each package defines its own input params.